### DISCUSSION TITLE
please help for the this warning msg: 'paramiko is not installed: No module named ''paramiko'''

### DISCUSSION THREAD

**Discussion by YuwanS** [semaphoreui/semaphore#1502] [Q&A]
i'm trying to run playbook to cisco router, but the warning is 
msg: 'paramiko is not installed: No module named ''paramiko'''
before than i already sure the SSH can connect form my control node (ubuntu) to cisco router.
and then i'm run manual with ansible-playbook command already success also. 

but with ansible semaphore the error is exist

**Comment by network-coa** [comment]
make sure paramiko is installed under the semahore user

**Comment by rogertheshruber** [comment]
when you run ansible from command line it looks for collections in /home/<USERNAME>/.ansible/collections/  when you run from semaphore it will tell you where it is looking for collections, look for a line like this in the output 
"ansible collection location = /tmp/semaphore/.ansible/collections:/usr/share/ansible/collections"
